content/posts/initial-http-api-is-released.md

```diff
@@ -0,0 +1,20 @@
+Title: Initial HTTP API is released
+Date: 2023-11-08 20:00
+Author: Kory Draughn
+Slug: initial-http-api-is-released
+Status: published
+
+
+The iRODS Consortium is pleased to announce the first release of the [iRODS HTTP API](https://github.com/irods/irods_client_http_api).
+
+It is released today as v0.1.0.
+
+It is an experimental redesign of the iRODS C++ REST API originally released in 2021. The primary goal of the project is to make it easy and safe for developers to interact with an iRODS zone. This falls right in line with the goals outlined in the [Consortium's roadmap for 2023](https://irods.org/roadmap).
+
+Users of the iRODS C++ REST API will find the iRODS HTTP API simpler to work with and more equipped for building rich applications and managing complex workloads. It also has support for OpenID Connect.
+
+A [Docker image](https://hub.docker.com/r/irods/irods_http_api) has been published on hub.docker.com for those interested in giving it a try.
```

Comment:
update
"has been published on hub.docker.com for those interested in giving it a try."
to 
"has also been published on Docker Hub for ease of deployment."

Reply:
Done.